src/reducers/search.js

```diff
@@ -12,6 +12,7 @@ export const initialState = {
     term: '',
     limit: 15,
     page: 1,
+    sortMethodValue: 'relevance',
```

Comment:
`sortMethodValue -> sortBy`
make the varible simple and clear.

Reply:
Finished